src/farcaster/getFarcasterUserAddresses.ts

```diff
@@ -0,0 +1,35 @@
+import { neynarGetCustodyAddressesForFid } from '../utils/neynar/user/neynarGetCustodyAddressForFid';
+import { neynarGetVerifiedAddressesForFid } from '../utils/neynar/user/neynarGetVerifiedAddressesForFid';
+import { FarcasterAddressType, GetFarcasterUserAddressesRequest } from './types';
+
+type GetUserAddressOptions =
+  | {
+      neynarApiKey?: string;
+    }
+  | undefined;
+
+/**
+ * Get the user address for a given fid
+ * @param fid  The frame id
+ * @param FarcasterAddressType represents whether the client wants a verified address or a custody address
+ * @returns the validation addresses or the custory address. If there is an error, it throws an error
+ */
+
+async function getFarcasterUserAddresses(
+  getFarcasterUserAddressRequest: GetFarcasterUserAddressesRequest,
+  options?: GetUserAddressOptions,
+): Promise<string | string[]> {
```

Comment:
Why have a string in one case a string array in an other? This feels like it will make the coding experience a bit more challenging as someone has to check what type is it before using it.

Reply:
Makes sense! I will update this to return string